draft-ietf-quic-recovery.md

```diff
@@ -519,6 +519,24 @@ bytes.
 Initial packets and Handshake packets may never be acknowledged, but they are
 removed from bytes in flight when the Initial and Handshake keys are discarded.
 
+### Speeding Up Handshake Completion
+
+When a server receives an Initial packet containing duplicate CRYPTO data,
+it can assume the client did not receive all of the server's CRYPTO data sent
+in Initial packets, or the client's estimated RTT is too small. When a
+client receives Handshake or 1-RTT packets prior to obtaining Handshake keys,
+it may assume some or all of the server's Initial packets were lost.
```

Comment:
What is the consequence of the client assuming server initials were lost? What would it do in this case to speed up the handshake?

Reply:
The client sending a packet can trigger this mechanism if the server implements it and it ensures the server is not blocked by the amplification factor.  I could add more detail in the second paragraph if it's helpful?

Reply:
Wait, do you mean "receives and successfully decrypts Handshake or 1-RTT", or I just get something that is marked as such?

Reply:
If you can decrypt them, then clearly the Initial packets you're waiting for weren't lost.